src/components/Todo/List.jsx

```diff
@@ -1,14 +1,14 @@
 import Item from "./Item";
 
 function List( props ) {
-  const { todoList } = props
+  const { todoList, onRemoveTodo } = props
 
   return (
     <div>
       {todoList && todoList.map(item => (
-        <Item key={item.id} todoItem={item} />
+        <Item key={item.id} todoItem={item} onRemoveTodo={() => onRemoveTodo(item.id)} />
```

Comment:
Thank you @pavel6767 for your review, I came up with passing the onRemoveTodo function down to the Item component and handled it there to prevent creating anonymous functions. My changes are in commit 972d98c.